main/webapp/modules/core/scripts/reconciliation/recon-dialog.html

```diff
@@ -24,7 +20,7 @@
 			<a class="button" bind="discoverServicesButton" href="https://reconciliation-api.github.io/testbench" target="_blank" ></a>
 		</div>
 		<div class="dialog-footer-bg">
-			<button class="button" bind="reconcileButton"></button>
+			<button class="button" style="font-weight: bold;" bind="nextButton"></button>
```

Comment:
Rather than manually applying a style here, I think it would be better to use the dedicated class for this: `button-primary`.
```suggestion
			<button class="button button-primary" bind="nextButton"></button>
```